docs/whats-new/preview-5.md

```diff
@@ -0,0 +1,595 @@
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+// required for the event processor client which will use the connectionName to get the connectionString.
+var blob = builder.AddAzureStorage("ehstorage")
+                  .AddBlobs("checkpoints");
+
+var eventHub = builder.AddAzureEventHubs("eventhubns")
+                      .AddEventHub("hub");
+
+builder.AddProject<Projects.EventHubsConsumer>("consumer")
+       .WithReference(eventHub)
+       .WithReference(blob);
+
+builder.AddProject<Projects.EventHubsApi>("api")
+       .WithExternalHttpEndpoints()
+       .WithReference(eventHub);
+
+builder.Build().Run();
+```
+
+As can be seen above the Event Hubs integration includes support for the Event Hubs processor architecture not just the consumer model (although either can be used). Refer to the README for the `Aspire.Azure.Messaging.EventHubs` package for examples of how to wire-up to the provisioned Event Hubs resource in your service projects.
+
+For more information, see [.NET Aspire Azure Event Hubs component](../messaging/azure-event-hubs-component.md).
+
+## Manifest changes
+
+The manifest format has been updated to support the new features in preview 5. The following sections outline the changes to the manifest format.
+
+### Volume in the manifest
+
+One of the features we have been wanted to add for some time is support
+for volumes in the manifest. Volumes are essential for some scale out scenarios
+around containers and for resiliency.
+
+Here is an example of a container in the manifest which defines multiple volumes:
+
+```json
+{
+    "type": "container.v0",
+    "image": "image/name:latest",
+    "volumes":
+    [
+      {
+          "name": "myvolume",
+          "target": "/mount/here",
+          "readOnly": false
+      },
+      {
+          "name": "myreadonlyvolume",
+          "target": "/mount/there",
+          "readOnly": true
+      },
+      {
+          "target": "/mount/everywhere",
+          "readOnly": false
+      }
+    ]
+}
+```
+
+It's up to the deployment tool that you are using to deploy the Aspire application to interpret these volume mounts and the technology that supports them. For example when using the Azure Developer CLI (`azd`) an Azure Storage account is created which exposes an Azure Files endpoint—and this is bound to the Azure Container App.
+
+A tool that targets Kubernetes might use Kubernetes' own concept of volumes, or one of the many storage providers that Kubernetes supports.
+
+### Endpoints
+
+We have expended the level of support for defining multiple endpoints in the manifest. To support this we added the `"port":` field to items in the `"bindings":` property on `container.v0` and `project.v0` resources. This port defines the exposed port that the target deployment environment will use when exposing the service. If not explicitly provided this port is assigned (in sequence) at manifest generation time.
```

Comment:
@IEvangelist I think we need a manifest update here on how Endpoints are affected now with the `WithExternalHttpEndpoints`

Before:
```json
"https": {
  "scheme": "https",
  "protocol": "tcp",
  "transport": "http"
},
```

After
```json
"https": {
  "scheme": "https",
  "protocol": "tcp",
  "transport": "http",
  "external": true
},
```